packages/context/src/context.ts

```diff
@@ -410,15 +411,20 @@ export class Context extends EventEmitter {
     if (!this.configResolver) {
       // First try the bound ConfigurationResolver to this context
       const configResolver = this.getSync<ConfigurationResolver>(
-        `${BindingKey.CONFIG_NAMESPACE}.resolver`,
+        ContextBindings.CONFIGURATION_RESOLVER,
         {
           optional: true,
         },
       );
       if (configResolver) {
+        debug(
+          'Custom ConfigurationResolver is loaded from %s.',
+          ContextBindings.CONFIGURATION_RESOLVER.toString(),
```

Comment:
Is it necessary to call `.toString()` here? Wouldn't `debug` call it automatically for `%s`?

Reply:
Here is my test:
```
debug('%s', {x: 1, toString: () => 'y'})
  x { x: 1, toString: [Function: toString] } +55s
```